### PR TITLE
Add two performance tests for SDS Vault CA flow

### PR DESCRIPTION
**Pull request by lei-tang**
Add two performance tests for SDS Vault CA flow.

### CODE REVIEW

**Comment by lei-tang** [comment]
https://github.com/istio/istio/issues/13439

File: perf/istio-install/values-istio-sds-vault.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    enabled: true
+    udsPath: "unix:/var/run/sds/uds_path"
+    useNormalJwt: true
+
+nodeagent:
+  enabled: true
+  image: node-agent-k8s
+  env:
+    # The IP address and the port number of a publicly accessible example Vault server.
+    CA_ADDR: "https://34.83.129.211:8200"
```
Comment (mandarjog) [inline perf/istio-install/values-istio-sds-vault.yaml]: Does this file have to repeat everything present in the main values.yaml?

You just enable and disable specific things right?

Comment (lei-tang) [reply perf/istio-install/values-istio-sds-vault.yaml]: This file has SDS and mTLS setting needed for the tests. These settings are explicately configured in this file to reduce the coupling with other values.yaml. The decoupling makes it less likely to have test breakage caused by the changes in other values.yaml.

**Comment by geeknoid** [comment]
@lei-tang Do we still want this PR?

**Comment by lei-tang** [comment]
@geeknoid Yes, this PR is stilled wanted for performance tests of SDS Vault CA flow.
@mandarjog Can you approve the PR?

**Comment by jacob-delgado** [comment]
Abandoned. Please reopen it if necessary.

**Pull request closed without merging**